articles/migrations/guides/calling-api-with-idtokens.md

```diff
@@ -0,0 +1,298 @@
+
+If you use auth0.js to access the Management API and manage your users, then your script will have to be updated.
+
+<div class="code-picker">
+  <div class="languages-bar">
+    <ul>
+      <li class="active"><a href="#coa-id-token" data-toggle="tab">Legacy (ID Token)</a></li>
+      <li><a href="#coa-access-token" data-toggle="tab">Current (Access Token)</a></li>
+    </ul>
+  </div>
+  <div class="tab-content">
+    <div id="coa-id-token" class="tab-pane active">
+      <pre class="text hljs">
+        <code>
+// get an ID token
+var webAuth = new auth0.WebAuth({
+  clientID: '${account.clientId}',
+  domain: '${account.namespace}',
+  redirectUri: '${account.callback}',
+  scope: 'openid',
+  responseType: 'id_token'
+});
+// create a new instance
+var auth0Manage = new auth0.Management({
+  domain: '${account.namespace}',
+  token: 'ID_TOKEN'
+});
+        </code>
+      </pre>
+      <div class="tab-pane-footer">
+        <ul>
+          <li>Asks for an ID Token in the response (<code>responseType: 'id_token'</code>)</li>
+          <li>Authenticates with the Management API using the ID Token</li>
+        </ul>
+      </div>
+    </div>
+    <div id="coa-access-token" class="tab-pane">
+      <pre class="text hljs">
+        <code>
+// get an access token
+var webAuth = new auth0.WebAuth({
+  clientID: '${account.clientId}',
+  domain: '${account.namespace}',
+  redirectUri: '${account.callback}',
+  audience: 'https://${account.namespace}/api/v2/˜',
+  scope: 'read:current_user',
+  responseType: 'token id_token'
+});
+// create a new instance
+var auth0Manage = new auth0.Management({
+  domain: '${account.namespace}',
+  token: 'ACCESS_TOKEN'
+});
+        </code>
+      </pre>
+      <div class="tab-pane-footer">
+        <ul>
+          <li>Asks for both an ID Token and an Access Token in the response (<code>responseType: 'token id_token'</code>)</li>
+          <li>Sets the Management API as the intended audience of the token (<code>audience: 'https://${account.namespace}/api/v2/˜'</code>)</li>
+          <li>Asks for the permission to read the current user's info (<code>scope: 'read:current_user'</code>)</li>
+          <li>Authenticates with the Management API using the Access Token</li>
+        </ul>
+      </div>
+    </div>
+  </div>
+</div>
+
+## Changes in Account Linking
+
+The changes in this functionality are the following:
+
+- You can no longer use an ID Token at the `Authorization` header
+- If you use an Access Token at the `Authorization` header, with `update:current_user_metadata` as the granted permission, then you must send the ID Token of the secondary account in the request's body
+- If you use an Access Token at the `Authorization` header, with `update:users` as the granted permission, then you must send the `user_id` of the secondary account in the request's body. The following must apply:
```

Comment:
If you use `update:users` then you get to chose between `user_id` or `id_token` in the payload. `update:users` is an admin scope, and admins can use their prefer method.